tests/e2e/pact/services/update-prescriptions.ts

```diff
@@ -204,6 +194,24 @@ export async function updatePrescriptions(
   })
 }
 
+function isRepeatDispenseNotification(messageHeader: fhir.MessageHeader) {
+  if (messageHeader.extension) {
+    const replacementOfExtension = getReplacementOfExtension(messageHeader.extension)
+    if (replacementOfExtension) {
+      return !!replacementOfExtension.valueIdentifier.value
+    }
+  }
```

Comment:
dont think it really matters but this function return is `void` if there isn't an extension, it should probably return `false` instead